docs/experimental-features.md

```diff
@@ -28,6 +28,19 @@ Enables local deployment capability. See [Bicep Local Providers](https://github.
 Moves defining extension configurations to the module level rather than from within a template. The feature also
 includes enhancements for Deployment stacks extensibility integration. This feature is not ready for use.
 
+### `onlyIfNotExists`
+The feature introduces the onlyIfNotExists decorator on a resource. The decorator will only deploy the resource if it does not exist.
```

Comment:
It might be worth adding a note to mention that this won't work until the backend service support has been deployed, just in case someone is eager to try it.